### ISSUE TITLE
Question about continuous action space

### ISSUE THREAD

**Issue by liuqianqian-x** [CORE-Robotics-Lab/MAGIC#4]
Hi authors,

Thanks for providing the code.

Can MAGIC be used in continuous action space? And how should I deal with parameters of "num_actions", "dim_actions" , "nactions", "actions_heads" and so on for continuous actions in your code [env_wrappers.py](https://github.com/CORE-Robotics-Lab/MAGIC/blob/af608fda8e31d6c9acd5a63f8b55db1f0cd04f12/env_wrappers.py#L33) and [action_utils.py](https://github.com/CORE-Robotics-Lab/MAGIC/blob/af608fda8e31d6c9acd5a63f8b55db1f0cd04f12/action_utils.py#L14) .

Looking forward to your guidance. Thanks!

**Comment by rohanpaleja27** [comment]
Hi,

Thanks for your question.

MAGIC can be extended to a continuous action space by 1) changing our loss function to support continuous action spaces and 2) updating the action heads to output a continuous value rather than a softmaxed discrete probability distribution. The rest of the framework can remain the same.

Some of the environment wrapper code supporting continuous action spaces is a remnant of much older code and may be deprecated. Since we only use 1 action_head (agents are sharing parameters), I would recommend creating a new flag for continuous and setting action_heads to 1. This should activate the statement in line 15 and lead to args.continuous=True. 

You may need to modify env_wrappers.py and/or your environment to support the action output from magic (which should be something like an input of size {1,num_agents})

Hope this helps!
